test/interval_tests/numeric.jl

```diff
@@ -390,3 +390,30 @@ end
     @test hull(v...) == II
     @test hull(v) == II
 end
+
+@testset "nthroot test" begin
```

Comment:
Could you add some tests for `BigFloat` as well?

Reply:
Added in `numeric.jl`